build/Dockerfile

```diff
@@ -1,4 +1,21 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
-COPY _output/bin/marker /marker
-COPY _output/bin/.version /.version
+ARG BUILD_ARCH=amd64
+FROM --platform=linux/${BUILD_ARCH} quay.io/centos/centos:stream9 AS builder
+RUN dnf install -y tar gzip jq && dnf clean all
+RUN ARCH=$(uname -m | sed 's/x86_64/amd64/') && \
+    GO_VERSION=$(curl -L -s "https://go.dev/dl/?mode=json" | jq -r '.[0].version') && \
+    curl -L "https://go.dev/dl/${GO_VERSION}.linux-${ARCH}.tar.gz" -o go.tar.gz && \
```

Comment:
lets take go version from go.mod please, as done on other network repos
easier to maintain

Reply:
Addressed in latest commit.